SplitFiles/control.c

```diff
@@ -4,6 +4,7 @@
 #include <arch/zx.h>
 #include <stdbool.h>
 
+#include "bread.h"
 #include "control.h"
 #include "game.h"
```

Comment:
This seems quite odd - control imports the header for game (which I think of as being the top level source file).  Feels like a bust with encapsulation/modularity.  Is there stuff in game that should be in control or even in bread?